DuckDuckGo/Subscription/ViewModel/SubscriptionEmailViewModel.swift

```diff
@@ -115,6 +115,7 @@ final class SubscriptionEmailViewModel: ObservableObject {
     }
     
     private func completeActivation() {
+        DailyPixel.fireDailyAndCount(pixel: .privacyProRestorePurchaseEmailSuccess)
```

Comment:
Should this be a daily pixel?

Reply:
from https://app.asana.com/0/1200019156869587/1205469378793604/f looks like so:

Restore Purchase using Email - Success
Type: Count pixel + Daily Pixel
Trigger: When a user successfully restores subscription using email address